.gitignore

```diff
@@ -24,4 +24,3 @@ test_tmp/
 # Coverage files
 .coverage
 htmlcov/
-ete
```

Comment:
@jhcepas is this line intended to be there?

Reply:
probably something from old times. I don't see why we need it now.